spec/models/reports/daily_follow_up_spec.rb

```diff
@@ -51,6 +51,9 @@
     expect(follow_ups.count).to eq(3)
     expect(follow_ups.with_hypertension.count).to eq(2)
     expect(follow_ups.with_diabetes.count).to eq(2)
+    expect(follow_ups.with_diabetes_only.count).to eq(1)
```

Comment:
Could match the blood pressure IDs here instead of matching counts to check if the right follow up is included in the right scope.

Reply:
Right. Will make the change before merging the PR